tests/widget/inline/widget-device-profile.test.js

```diff
@@ -0,0 +1,16 @@
+import { expect, test } from '@playwright/test';
+
+import { asyncEvents } from '../../utilities/async-events.js';
+
+test('Inline widget with login in Canadian French', async ({ page, context }) => {
+  const { clickButton, navigate } = asyncEvents(page);
+  await navigate('widget/inline?journey=DeviceProfileNode');
+  await page.getByRole('textbox', { name: 'Username' }).type('demouser');
+  await page.getByRole('textbox', { name: 'Password' }).type('j56eKtae*1');
+  await clickButton('Next', '/authenticate');
+  await page.waitForRequest(/\/authenticate/, async () => {
+    const text = page.getByText('Collecting Profile');
+    await expect(text).toBeVisible();
+  });
```

Comment:
I believe this is redundant as the `clickButton` already accounts for waiting until the request is resolved. Once `clickButton` itself resolves, you can test whatever you expect to be in the next view.

Reply:
hmm idk this was required for me to get it working.

Reply:
The device profile text disapeared too quickly.

Reply:
I've pulled this in locally, and I believe I know what's happening. This block of code is never executed. So, the assertion is never applied, which gives you a false sense of "passing". The reason for this is the `/authenticate` request is already resolved within the `clickButton` method, which is awaited, so the following `waitForRequest` event listener is never triggered. If you write a console message within it, you'll never see it logged.

The reason it doesn't work when it's not wrapped in this `waitForRequest` is because the "Collecting Profile" message is never displayed by the UI, ever. It's not that it disappears to quickly. You can see that the message is missing in your GIF provided in the PR description.

That's due to the fact that the node in the journey uses `message` as the key. Keys have to be locales or languages, so the key would be `en` or `en_US` with the value of `"Collecting Profile"`. Once you change the key to `en` you can then directly assert using the below without wrapping it in a `waitFor*`:

```
await expect(page.getByText('Collecting Profile')).toBeVisible();
```